External/TestDataSet/rawdata/sub-Sub1/perf/sub-Sub1_aslcontext.tsv

```diff
@@ -0,0 +1,2 @@
+volume_type
+
```

Comment:
has to be filled with `control` `label` etc - one line per each and the same number as number of volumes.

Reply:
So we should have 60 volumes, that means 30 acquisition pairs. I don't know if it's control/label or label/control. Probably control/label, right? 9882645